### DISCUSSION TITLE
Enabling `RankDisplay` on break screen

### DISCUSSION THREAD

**Discussion by fyyrraaa** [ppy/osu#23041] [Ideas] [answered]
From what I remembered, the break info screen used to have a rank display alongside the currently-visible grade and accuracy displays. From #15185 I understand that at the time since it wasn't displaying anything so it was hidden with the intent of revisiting it if the break screen has any design upgrades. 

With the leaderboard and score submission being enabled for lazer, it would be nice to add them back in again. However, it would be understandable if this is not desirable and is not the intended direction.

Relevant pull request: #15186

**Comment by peppy** [comment, marked as answer]
The intention is definitely to bring it back.
